internal/utils/utils.go

```diff
@@ -27,7 +28,12 @@ const (
 
 var (
 	// UnixEpoch is the number of seconds that have elapsed since January 1, 1970
-	UnixEpoch = time.Unix(0, 0)
+	UnixEpoch          = time.Unix(0, 0)
+	DefaultClient      = &http.Client{Transport: &http.Transport{Proxy: http.ProxyFromEnvironment}}
+	InsecureHttpClient = &http.Client{Transport: &http.Transport{
```

Comment:
I would remove Http for simplicity
```suggestion
	InsecureClient = &http.Client{Transport: &http.Transport{
```

Reply:
@jotadrilo ok. Done that